spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ApplicationHelper do
+  include ApplicationHelper
+  include ViewHelper
+  include GovukComponentsHelper
+  include GovukLinkHelper
```

Comment:
It is making it convoluted.

https://relishapp.com/rspec/rspec-rails/docs/helper-specs/helper-spec

Reply:
i think this the whole spec is a bit weird. It was written as a feature spec and it's setup to render views/markup. I think it would make more sense to refactor post migration as it will need to be re-written.